README.md

```diff
@@ -288,3 +288,65 @@ To fix this, follow these steps:
     You will again see a warning prompt saying "Running "Android Emulator" will harm your computer"
     Click on open
     ```
+
+
+# 🔧 Firebase Configuration
+
+To ensure Firebase works correctly with your app, follow the steps below to match the app's identifiers with the ones registered in Firebase.
+
+---
+
+## ✅ Android Setup
+
+### 1. Verify Package Name
+
+- Open your `google-services.json` file.
+- Locate the `package_name` field:
+  ```json
+  "package_name": "com.example.yourapp"
```

Comment:
How would this work where the identifier changes with build, ex [here](https://github.com/jalantechnologies/react-native-template/pull/148)